algebird-test/src/test/scala/com/twitter/algebird/ApplicativeProperties.scala

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package com.twitter.algebird
 
 import org.scalatest.{Matchers, PropSpec}
-import org.scalatest.prop.PropertyChecks
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
```

Comment:
I guess we are not using this import since it doesn’t appear below.

Reply:
Ah nice catch! I should have double import usage.